cost/low_service_usage/low_service_usage.pt

```diff
@@ -0,0 +1,255 @@
+        "start_at": start_at,
+        "end_at": end_at
+      },
+      headers: {
+        "Api-Version": "1.0",
+        "User-Agent": "RS Policies",
+      }
+    }
+  EOS
+end
+
+datasource "ds_format_costs" do
+  run_script $js_format_costs, $ds_new_bc_costs, $ds_cloud_vendor_accounts, $param_include_unallocated
+end
+
+script "js_format_costs", type: "javascript" do
+  parameters "new_bc_costs", "ds_cloud_vendor_accounts", "include_unallocated"
+  result "formatted_data"
+  code <<-EOS
+
+  var type = 'cost_amortized_unblended_adj';
+
+  var formatted_data = [];
+  var bcs = [];
+
+  _.each(new_bc_costs, function(bcc){bcs.push(bcc.id)})
+  bcs = _.uniq(bcs) ;
+
+  _.each(bcs, function(bc_id){
+    var new_bc_cost_objs = _.reject(new_bc_costs, function(new_bc_cost){ return new_bc_cost.id != bc_id });
+    if ( bc_id != 'unallocated' ){
+      _.each(new_bc_cost_objs, function(new_bc_cost_obj){
+        if (new_bc_cost_obj.vendor == "AWS" ) {
+          var account = _.reject(ds_cloud_vendor_accounts, function(ds_cloud_vendor_account){ return ds_cloud_vendor_account.id != new_bc_cost_obj.vendor_account_name });
+          var vendor_account_name = account[0].name
+        } else {
+          var vendor_account_name = new_bc_cost_obj.vendor_account_name
+        }
+        var new_sum = parseFloat(Math.round(new_bc_cost_obj.cost_amortized_unblended_adj*100)/100)
+        if ( new_sum != 0 ) {
+          formatted_data.push({
+            name: new_bc_cost_obj.name,
+            id: bc_id,
+            service: new_bc_cost_obj.service,
+            new_sum: new_sum,
+            vendor: new_bc_cost_obj.vendor,
+            vendor_account_name: vendor_account_name
+          })
+        }
+      })
+    }
+    if ((bc_id == 'unallocated') && (include_unallocated == "false")) {
+      return
+    } else {
+      _.each(new_bc_cost_objs, function(new_bc_cost_obj){
+        if (new_bc_cost_obj.vendor == "AWS" ) {
+          var account = _.reject(ds_cloud_vendor_accounts, function(ds_cloud_vendor_account){ return ds_cloud_vendor_account.id != new_bc_cost_obj.vendor_account_name });
+          var vendor_account_name = account[0].name
+        } else {
+          var vendor_account_name = new_bc_cost_obj.vendor_account_name
+        }
+
+        var new_sum = parseFloat(Math.round(new_bc_cost_obj.cost_amortized_unblended_adj*100)/100)
+        if ( new_sum != 0 ) {
+          formatted_data.push({
+            name: new_bc_cost_obj.name,
+            id: bc_id,
+            service: new_bc_cost_obj.service,
+            new_sum: new_sum,
+            vendor: new_bc_cost_obj.vendor,
+            vendor_account_name: vendor_account_name
+          })
+        }
+      })
+    }
+  })
+EOS
+end
+
+escalation "esc_budget_alert" do
+  email $param_email
+end
+
+policy "policy_budget_alert" do
+  validate_each $ds_format_costs do
+    summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): Low Service Usage Detected"
```

Comment:
Optima APIs are at the Org level, the summary template should be updated to reflect the org id and name.